Arrow + Prefab changes

Shadowth117 opened the pull request.

Shadowth117 committed:
Arrow + Prefab changes

Shadowth117 committed:
ArrowFixAgain

Gman8r reviewed (changes requested):
Fix the runtime bugs and migrate code to not use GameObject.find to fix that one bug

Shadowth117 committed:
Exception,update, collider fix

Upped size of collider, fixed exception (hopefully), fixed referencing to occur outside of update loops.
